src/components/member-card/super-user-options/index.js

```diff
@@ -7,6 +7,7 @@ const SuperUserOptions = ({ showSettings, username }) => {
 
   const showModal = (e) => {
     e.preventDefault();
+    e.stopPropagation();
```

Comment:
is the need of both `preventDefault` and `stop propagation` needed here?

Reply:
It is done by @prakashchoudhary07 https://github.com/Real-Dev-Squad/website-members/pull/513/commits/336e4b43fbc16f33e6a89795963da55bb57aaea3